### PR TITLE
Restructure cloud integrations

### PR DESCRIPTION
**Pull request by jaffrepaul**
Small iteration to better organize content. 

Group integration pages, update URLs & redirects. 

----

Associated `on` link updates: https://github.com/cypress-io/cypress-services/pull/5628

### CODE REVIEW

File: docs/guides/cloud/integrations/bitbucket-integration.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Bitbucket Integration
-sidebar_position: 90
+sidebar_position: 10
```
Comment (elylucas) [inline docs/guides/cloud/integrations/bitbucket-integration.mdx:3]: we should sort these nav items the same order they were in the initial nav. IE, GitHub, GitLab, Bitbucket, etc...

File: docs/guides/cloud/integrations/github-integration.mdx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: GitHub Integration
-sidebar_position: 70
+sidebar_position: 20
+sidebar_label: GitHub
```
Comment (elylucas) [inline docs/guides/cloud/integrations/github-integration.mdx:4]: for better URLs, we should either rename the md files to take out the '-integration' (so we don't repeat the word integration in the URL), or use the slug meta data item to specify a different URL.

File: netlify.toml [hunk 3]
```diff
@@ -385,7 +385,7 @@
 
 [[redirects]]
```
Comment (elylucas) [inline netlify.toml:386]: we'll need to add net new redirects to forward "/guides/xxx-integration" to "/guides/integration/xxx"

**Review by elylucas — CHANGES_REQUESTED** [review]
Few change requests to look at

File: docs/guides/cloud/integrations/bitbucket-integration.mdx (hunk 1 above)
Comment (jaffrepaul) [reply docs/guides/cloud/integrations/bitbucket-integration.mdx:3]: I set them per the chat about analytics for these pages

File: netlify.toml [hunk 4]
```diff
@@ -410,5 +440,5 @@
 [[plugins]]
 package = "@netlify/plugin-sitemap"
 
-  [plugins.inputs]
-  baseUrl = "https://docs.cypress.io"
+[plugins.inputs]
+baseUrl = "https://docs.cypress.io"
```
Comment (pstakoun) [inline netlify.toml:444]: this an intentional change?

**Review by elylucas — APPROVED** [review]

**Pull request merged**